package/setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+author = Screwdriver.cd team
+description-file = ../README.md
+name = screwdriver_cd_setup
+url = https://git.ouroath.com/dhubbard/in-a-box
```

Comment:
https://git.ouroath.com/screwdriver-cd/in-a-box?

Reply:
Good catch